src/inheritAttributes.js

```diff
@@ -287,6 +287,45 @@ export const parseCaptionServiceMetadata = (service) => {
   }
 };
 
+/**
+ * A map callback that will parse all event stream data for a collection of periods
+ * DASH ISO_IEC_23009 5.10.2.2
+ * https://dashif-documents.azurewebsites.net/Events/master/event.html#mpd-event-timing
+ *
+ * @param {PeriodInformation} period object containing necessary period information
+ * @return a collection of parsed eventstream event objects
+ */
+export const toEventStream = (period) => {
+  // get and flatten all EventStreams tags and parse attributes and children
+  return flatten(findChildren(period.node, 'EventStream').map((eventStream) => {
+    const eventStreamAttributes = parseAttributes(eventStream);
+    const schemeIdUri = eventStreamAttributes.schemeIdUri;
+    // schemeIdUri is mandatory for EventStream tags
+
+    if (!schemeIdUri) {
+      return;
+    }
```

Comment:
thought(non-block): I wonder if it might be better to still return the eventstream even without the scheme. That way, we're more permissive. Maybe log a warning, but still parse/return it?


-----------------------------
nit:
```suggestion

    // schemeIdUri is mandatory for EventStream tags
    if (!schemeIdUri) {
      return;
    }
```

Reply:
Removing the early return for `schemeIdUri` for now. I agree that we should make it more permissive is possible. I'm going to leave out a warning log as well, since we're currently not utilizing the `schemeIdUri` to determine the parsing outcome, it seems we shouldn't warn unless we have a reason to. For example, if we implement a specific parsing scheme here or downstream and the provided `schemeIdUri` is `undefined` or not currently supported.